### ISSUE TITLE
Test Net 

### ISSUE THREAD

**Issue by sinash14** [CyberPunkMetalHead/binance-trading-bot-new-coins#37]
Hi,

Thanks for sharing the codes. I am trying to run the codes on Binance test net but I am getting below error: 

requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.binance.binance_tld', port=443): Max retries exceeded with url: /api/v3/ping (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f12e8610760>: Failed to establish a new connection: [Errno -2] Name or service not known'))

Could you please assist ? Thanks

**Comment by mevljas** [comment]
You have to set binance_tld property in the auth.yml file.

**Comment by mevljas** [comment]
Have you pulled the latest changes?

**Comment by sinash14** [comment]
Yes,  I started frm the scratch again. This time I created API from my binance account not test net. However, I am getting below error now:

requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.binance.binance_tld', port=443): Max retries exceeded with url: /api/v3/ping (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7fe47664ccd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

**Comment by easyguyme** [comment]
> Yes, I started frm the scratch again. This time I created API from my binance account not test net. However, I am getting below error now:
> 
> requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.binance.binance_tld', port=443): Max retries exceeded with url: /api/v3/ping (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7fe47664ccd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

You need to update your auth file and change binance_tld: "BINANCE_TLD" to binance_tld: "COM" or binance_tld: "US" depending with the country you are in.

**Comment by sinash14** [comment]
Great! that resolved it! Thank you!

**Comment by easyguyme** [comment]
@sinash14 @CyberPunkMetalHead please close this issue
